packages/template-mrt-reference-app/app/ssr.js

```diff
@@ -80,14 +76,16 @@ const ENVS_TO_EXPOSE = [
     'tz'
 ]
 
-const HEADERS_TO_REDACT = ['x-api-key', 'x-apigateway-context', 'x-apigateway-event']
-
```

Comment:
@kieran-sf Would this be considered a breaking change?

If customers were relying on the removal of those headers and suddenly found them present in the requests, Could this lead to unexpected behaviour, errors or data exposure?

At the very least, are we planning on documenting the changes that could impact customers and provide alternative methods for those who need to maintain the current behaviour?